arrays.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>lesson 7 arrays</title>
+</head>
+
+<body>
+<script src="https://travisshears.com/apps/random-wiki/0.0.4--alpha3/src.7b19814e.js"></script>
+<h2 id="firstA" style=color:blue>The first article</h2>
+<h2 id="lastA" style=color:red>The last article</h2>
+<ul id="articlesList"><li></li></ul>
+
+<script>
+    let articles = [];
+    const firstAEl = document.getElementById('firstA');
+    const lastAEl = document.getElementByID('lastA');
```

Comment:
wrong name of a function should be like that:
`document.getElementById('lastA');`

Reply:
Hi Ivan,
Thanks for all your comments, VERY HELPFUL in my state of tiredness :) (teaching my three kids at home). 
I fixed the other issues you mentioned, but I just didn't get this one:
I have document.getElementByID('lastA') there, and you said it should be exactly that. So I'm not getting whether I should remove something from the code, or where is the problem. Thanks a lot for your patience :)
Stanka

Reply:
What I mean is that in your code you have
`document.getElementByID('lastA');`
but it should be 
`document.getElementById('lastA');`
You wrote **ID** in capital letters but it's only first capital **I** and **d** is a small one.